docs/developers-guide/sink-targets/substreams-sink-clickhouse.md

```diff
@@ -0,0 +1,93 @@
+---
+description: Aleno Substreams ClickHouse sink
+---
+
+# [`Substreams`](https://substreams.streamingfast.io/) [Clickhouse](https://clickhouse.com/) sink module
+
+
+> [`substreams-sink-clickhouse`](https://github.com/Aleno1/substreams-sink-clickhouse) is a tool that allows developers to pipe data extracted metrics from a blockchain into a ClickHouse DBMS for warehousing purposes.
+
+> This sink is very similar to the [substreams-sink-postgres](https://github.com/streamingfast/substreams-sink-postgres) and one could refer to its [docs](./substreams-sink-postgres.md) to understand its concepts.
+
+
+### Quickstart
+
+1. Install `substreams-sink-clickhouse` by using the pre-built binary release [available in the releases page](https://github.com/Aleno1/substreams-sink-clickhouse/releases). Extract `substreams-sink-clickhouse` binary into a folder and ensure this folder is referenced globally via your `PATH` environment variable.
+
+    > **Note** Or install from source directly `go install github.com/Aleno1/substreams-sink-clickhouse/cmd/substreams-sink-clickhouse@latest`.
+
+1. Start Docker Compose:
+
+    ```bash
+    docker compose up
+    ```
+
+    > **Note** Feel free to skip this step if you already have a running ClickHouse instance accessible, don't forget to update the connection string in the command below.
+
+2. Setup ClickHouse
+
+    Connect to ClickHouse
+
+    ```bash
+    docker compose exec ch_server clickhouse-client -u dev-node --password insecure-change-me-in-prod -h localhost
+    ```
+
+    And create necessary tables to run the sink
+
+    ```sql
+        CREATE TABLE block_meta
+    (
+        id          String, 
+        at          String,
+        number      Int32,
+        hash        String,
+        parent_hash String,
+        timestamp   String,
+        PRIMARY KEY (id),
+    )
+    ENGINE = MergeTree()
+    ORDER BY id;
+
+    CREATE TABLE cursors
+    (
+        id         String,
+        cursor     String,
+        block_num  Int64,
+        block_id   String,
+        PRIMARY KEY (id)
+    ) ENGINE = MergeTree()
+    ORDER BY id;
+    ```
+
+    > **Note** Each create table query must be run independently as ClickHouse doesn't support multiple create table queries at once.
+
+3. Run the sink
+
+    Use the precompiled Ethereum Block Meta [substreams](https://github.com/streamingfast/substreams-eth-block-meta/releases/download/v0.4.1/substreams-eth-block-meta-v0.4.1.spkg)
```

Comment:
Latest is `https://github.com/streamingfast/substreams-eth-block-meta/releases/download/v0.4.3/substreams-eth-block-meta-v0.4.3.spkg` and uses the latest canonical package id for database changes